app/src/main/java/org/kiwix/kiwixmobile/main/KiwixWebViewClient.java

```diff
@@ -106,11 +112,34 @@ public void onPageFinished(WebView view, String url) {
 
   private void inflateHomeView(WebView view) {
     LayoutInflater inflater = LayoutInflater.from(view.getContext());
-    home = inflater.inflate(R.layout.content_main, view, false);
+    if (KiwixApplication.getInstance().getResources().getConfiguration().orientation ==
```

Comment:
It is not so difficult to use a landscape layout in android. You can read this to get it done easily: https://stackoverflow.com/questions/2124046/how-do-i-specify-different-layouts-for-portrait-and-landscape-orientations.